lib/build.js

```diff
@@ -122,6 +124,17 @@ function build (gyp, argv, callback) {
   /**
    * Search for the location of "msbuild.exe" file on Windows.
    */
+  function findMsbuild15() {
+    log.verbose('looking for VS2017 msbuild')
+    var msbuild_path = findVS.locateMsbuild()
```

Comment:
Errors finding VS2017 should be ignored, it should still be able to find other versions. Something like:

```js
    try {
      var msbuild_path = findVS.locateMsbuild()
    } catch (_) {}
```